packages/tokens/src/bin/Generators/SassGenerator.ts

```diff
@@ -28,12 +29,23 @@ function toSassVariable(varName: string, value: any, isDefault = true) {
 }
 
 function JSONToSassMap(json: Object, isDefault = true) {
+  function toSassValue(value: any) {
+    if (typeof value === "object") {
+      return deepQuoteObjectKeys(value);
+    }
+    if (typeof value === "string") {
+      // If we have value = Font 1, Font 2 it will break the map so we need to quote it.
+      if (value.indexOf(",") >= 0) {
+        return `'${value}'`;
+      }
+    }
```

Comment:
I'm afraid that will not work with some values (e.g: `rgb`, `hsla`, etc.) ?

Reply:
Yep but we will have to use the interpolation operator .. 😕 or a more complicated solution would be to create isolated variable and then assign them to the map, but it would be more tedious, my idea:

( we would have to make an automatic var generation, keep me tuned )

```
$font: Arial, Roboto, Whatever
$themes: (
    'font': $font
)
```

Reply:
Or we should use array for this kind of values ? 🤔